installer/scripts/service_control

```diff
@@ -328,6 +331,15 @@ start_all_omsagent()
        setup_variables ${ws_id}
        start_omsagent
     done
+
+    for ws_id in $NON_OMS_WS
+    do
+        ls -1 $ETC_DIR | grep -w scom > /dev/null 2> /dev/null
```

Comment:
Ditto to above (for `ls -l`). Also, do you want to match on `scom` here? This looks like a generalized routine, wouldn't you want to match on `${ws_id}` instead of `scom`?